provider/aws.go

```diff
@@ -282,6 +282,16 @@ func newChanges(action string, endpoints []*endpoint.Endpoint) []*route53.Change
 	return changes
 }
 
+func getTTLValue(ep *endpoint.Endpoint) *int64 {
```

Comment:
This function should not be needed, if you need it, it should be a function receiver of type endpoint.Endpoint